webapp/src/components/AssetTransactionHistory/AssetTransactionHistory.js

```diff
@@ -39,21 +61,21 @@ export default class AssetTransactionHistory extends React.PureComponent {
         <Link to={locations.profilePageDefault(address)}>
           <AddressBlock
             address={address}
-            scale={4}
+            scale={3.4}
             hasTooltip={false}
             hasLink={false}
           />&nbsp;
-          <span className="short-address">{shortenAddress(address)}</span>
+          <span className="short-address">{shortenOwner(address)}</span>
```

Comment:
Why are we doing this? is it a CSS problem? I think it can be avoided changing the column sizes:

![image](https://user-images.githubusercontent.com/692077/52284460-b7491800-2964-11e9-9d6c-61d79f3cd59d.png)


that fixes it, keeping shortenAddress